--- a/gitconfig
+++ b/gitconfig
@@ -1,6 +1,7 @@
 [user]
     name = William Stevens
     email = contact@wastevensv.com
+    signingkey = 346501FAB7B4DB6B
 [push]
     default = current
 [color]
@@ -11,3 +12,7 @@
     autosetuprebase = always
 [pull]
     rebase = true
+[commit]
+    gpgsign = true
+[gpg]
+    program = gpg2-wrapper
--- /dev/null
+++ b/bin/gpg2-wrapper
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/gpg2 --batch --no-tty "$@"
